html/modules/custom/ocha_assessments/component/src/ocha-assessments-map.js

```diff
@@ -11,8 +11,9 @@ class OchaAssessmentsMap extends OchaAssessmentsBase {
     return [
       super.styles,
       css`
-        :host { display: block;
-          border: 1px solid green;
+        :host {
+          display: block;
+          border: 1px solid transparent;
```

Comment:
Feel free to remove the border completely

Reply:
When I removed the border declaration, it appears it defaults to 1px black so I needed to leave this in to override it.